demisto_sdk/commands/common/update_id_set.py

```diff
@@ -1881,14 +1881,9 @@ def has_duplicate(id_set_subset_list, id_to_check, object_type=None, print_logs=
         if object_type == 'Layouts':
             if dict1.get('kind', '') != dict2.get('kind', ''):
                 return False
-            if dict1.get('typeId', '') != dict2.get('typeId', ''):
+            if dict1.get('typeID', '') != dict2.get('typeID', ''):
                 return False
 
-        # If they have the same pack name they actually the same entity.
-        # Added to support merge between two id-sets that contain the same pack.
```

Comment:
will this be possible as a legitimate case? or should it always be considered as a duplicate?

Reply:
This comment refers to the original condition created by Chanoch. We had that meeting bach then when we decided that it should be considered as a duplicate.